### ISSUE TITLE
PKG-8 not detected

### ISSUE THREAD

**Issue by dewhattens** [openpreserve/odf-validator#73]
Put extra filed in mimetype

Testing D:\ODS1\TestCorpusV2\PKG-8\ExtraFieldInMimeType.ods
APP-1: [INFO] Validating D:\ODS1\TestCorpusV2\PKG-8\ExtraFieldInMimeType.ods.
APP-4: [INFO] Validation report for D:\ODS1\TestCorpusV2\PKG-8\ExtraFieldInMimeType.ods.
DOC-1: D:\ODS1\TestCorpusV2\PKG-8\ExtraFieldInMimeType.ods [ERROR] An OpenDocument document SHALL be a single XML file or a Zip file.
XML-3: D:\ODS1\TestCorpusV2\PKG-8\ExtraFieldInMimeType.ods [ERROR] Not a well formed XML document. XML parsing exception at line 1 and column 1: Content is not allowed in prolog..
NOT VALID, 2 errors, 0 warnings and 0 info messages.

**Comment by carlwilson** [comment]
This now appears to be working for the latest builds and the corpus file in question:

```bash
APP-1: [INFO] Validating ExtraFieldInMimeType.ods. 
APP-4: [INFO] Validation report for ExtraFieldInMimeType.ods. 
PKG-6: mimetype [ERROR] The "mimetype" file SHALL NOT be compressed. 
PKG-8: mimetype [ERROR] The “mimetype” file SHALL NOT use an 'extra field' in its header. 
DOC-3: mimetype [INFO] OpenDocument MIMETYPE extrafield
application/vnd.oasis.opendocument.spreadsheet detected 
PKG-12: META-INF/manifest.xml [ERROR] An OpenDocument package mimetype file content SHALL be equal to the manifest:media-type attribute of the manifest <manifest:file-entry> element whose manifest:full-path attribute has the value "/". 
NOT VALID, 3 errors, 0 warnings and 1 info messages. 
```

Please confirm and close @dewhattens
